src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -4184,6 +4184,66 @@ namespace winrt::TerminalApp::implementation
         }
     }
 
+    winrt::fire_and_forget TerminalPage::ActionSaved(winrt::hstring input, winrt::hstring name, winrt::hstring keyChord)
```

Comment:
Why put these two functions here instead of having them next to the related code inside `AppActionHandlers.cpp`? I feel like it should either be all here or all there.

Reply:
updated